src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -94,9 +94,10 @@ public void parseCommand_list() throws Exception {
     @Test
     public void parseCommand_sort() throws Exception {
         List<String> keywords = Arrays.asList("foo", "bar", "baz");
-        SortCommand command = (SortCommand) parser.parseCommand(
-                SortCommand.COMMAND_WORD + " " + keywords.stream().collect(Collectors.joining(" ")));
-        assertEquals(new SortCommand(new PersonHasTagPredicate(TestUtil.stringsToTags(keywords))), command);
+        FilterCommand command = (FilterCommand) parser.parseCommand(
+                seedu.address.logic.commands.FilterCommand.COMMAND_WORD + " "
```

Comment:
Eh how come become like this?
```suggestion
                FilterCommand.COMMAND_WORD + " "
```